src/domain/actuator.py

```diff
@@ -0,0 +1,34 @@
+"""Actuator module"""
+
+import logging
+from abc import abstractmethod
+from enum import Enum
```

Comment:
from abc import abstractmethod
from enum import Enum
import logging

Reply:
Antionio will resolve this as part of his rebase of this changeset.